types/architect__utils/updater.d.ts

```diff
@@ -0,0 +1,18 @@
+export {};
+type UpdaterMethods = Record<
+    | "start"
+    | "status"
+    | "done"
+    | "cancel"
+    | "err"
+    | "warn"
+    | "raw"
+    | "update"
+    | "stop"
+    | "error"
+    | "fail"
+    | "warning",
+    (...args: unknown[]) => void
+>;
+export type Updater = UpdaterMethods & Record<"verbose" | "debug", UpdaterMethods>;
+export function updater(name: string): Updater;
```

Comment:
This is just one of the exports, but the package seems to document many more. Is this intentional?

I would probably suggest writing them out so that they can be used, even if they are `any` or something.

Reply:
Yes. The interface is very large, and not much of it is documented. This is just a start at typing it.